apps/model/model.js

```diff
@@ -730,7 +740,8 @@ function uploadModel() {
 }
 
 async function deleteModel(name) {
-  if (confirm('Are you sure you want to delete this model?')) {
+  modelName = name.split('/').pop().split('_').splice(2).join('_').slice(0, -3);
```

Comment:
```  modelName ``` is the global array which stores the names of the model for the ``` checkDuplicateNameFunction ``` . I think you should use a different variable name in line 743 or else the array is reinitialized wrongly every time ``` deleteModel ``` is called . Am  I missing something @mgautam98 ?

Reply:
Yes, you are right. A different name should have been used I missed it. It will not cause any problem because after deleting the modelName will be reinitialized so no a problem. But It is confusing and I will correct it in next PR or you can do it too if you want in you next PR.